docs/en/log-retention-policy.rst

```diff
@@ -0,0 +1,55 @@
+Wallet Providers, Credential Issuers, and Relying Parties must implement appropriate security measures.
+
+Logs related to Wallet data exchange activities (accesses, transactions, credential issuance/revocation) concerning the End User, as the data subject, MUST be retained for a limited period for security, fraud prevention, dispute resolution, and legal obligations.
+
+
+In accordance with the following general principles and in full compliance with them, the retention of logs and personal data in digital identity wallets must adhere to the principles outlined in the GDPR (Article 5), specifically:
+
+- **Lawfulness, Fairness, and Transparency** (Article 5, Paragraph 1, Letter a): Personal data MUST be processed lawfully, fairly, and in a transparent manner in relation to the data subject.
+- **Purpose Limitation** (Article 5, Paragraph 1, Letter b): Personal data MUST be collected for specified, explicit, and legitimate purposes and not further processed in a manner that is incompatible with those purposes.
+- **Data Minimization** (Article 5, Paragraph 1, Letter c): Personal data MUST be adequate, relevant, and limited to what is necessary in relation to the purposes for which they are processed.
+- **Storage Limitation** (Article 5, Paragraph 1, Letter e): Personal data MUST be kept in a form which permits identification of data subjects for no longer than is necessary for the purposes for which the personal data are processed.
+- **Integrity and Confidentiality** (Article 5, Paragraph 1, Letter f): Personal data MUST be processed in a manner that ensures appropriate security, including protection against unauthorized or unlawful processing and against accidental loss, destruction, or damage, using appropriate technical or organizational measures.
+
+Wallet Providers, Credential Issuers, and Relying Parties MUST implement appropriate technical and organizational measures to ensure the security of logs and personal data, in compliance with Article 32 of the GDPR.
+
+Unless specific legal obligations dictate otherwise, and with the definition of sector-specific regulation defining appropriate motivations, the maximum retention period for data logs is 24 months. Logs MUST be securely stored to ensure integrity and immutability.
+
+Wallet Provider Log Retention Policy
+------------------------------------
+
+Personal data related to the registration and management of Wallet Instances can be retained for up to 24 months after the deactivation/revocation of the Wallet or the associated User account, unless legal obligations require longer retention.
+
+Credential Issuer Log Retention Policy
+--------------------------------------
+
+Credential Issuers define the retention periods for Credentials based on sector-specific regulations. In the absence of specific regulations, the retention period for credentials should not exceed 5 years from the date of issuance.
```

Comment:
This period should take into account the time validity period of a Digital Credential. Suppose that a Credential has a validity period greater than 5 years (I know, currently this is not our case) this retention policy could raise some problems.

Reply:
I need to include ISO 27001 and further details about what needs to be logged (signed tokens, not necessarly with their disclosures) and so on ...

Reply:
I would remove all these thresholds making a normative base set to 12 months only and give refs to any sectorial-specific regulations